### PR TITLE
objc: support string arguments to Send/SendSuper

### PR DESCRIPTION
**Pull request by TotallyGamerJet**

### CODE REVIEW

File: objc/objc_runtime_darwin_test.go [hunk 1]
```diff
@@ -20,42 +21,62 @@ func init() {
...
+		fmt.Println("In Child")
+		self.SendSuper(_cmd)
+	}), "v@:\x00")
+	child.Register()
+
+	objc.ID(child).Send(objc.RegisterName("new\x00")).Send(objc.RegisterName("doSomething\x00"))
+
+	// Output: In Child
+	// In Super!
+}
```
Comment (hajimehoshi) [inline objc/objc_runtime_darwin_test.go:82]: Option: what about not using \x00 in the examples?

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

File: objc/objc_runtime_darwin_test.go (hunk 1 above)
Comment (TotallyGamerJet) [reply objc/objc_runtime_darwin_test.go:82]: Sure!

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**